doc/release-notes/whats-new-3.22.md

```diff
@@ -16,19 +16,19 @@ Rapid Impact Assessment platform
 --------------------------------
 
 Most of the work went into our internal platform for Rapid Impact
-Assessment. There is a single codebases with three different interfaces
+Assessment. There is a single codebase with three different interfaces
 depending on the kind of user:
 
-- level 0 users: can only see computations performed by other user and shared
-- level 1 users: can only perform ShakeMap calculations determined by a ShakeMap ID
+- level 0 users: can only see results of computations performed by other users and published
```

Comment:
Here Paul wants to use the name "shared" and not "published" so I will keep that.